src/components/icon/js/iconDirective.js

```diff
@@ -182,9 +182,15 @@ function mdIconDirective($mdIcon, $mdTheming, $mdAria ) {
    */
   function postLink(scope, element, attr) {
     $mdTheming(element);
+    var lastIcon = scope.fontIcon;
+    var lastSet = $mdIcon.fontSet(scope.fontSet);
 
     prepareForFontIcon();
 
+    attr.$observe('mdFontIcon', fontIconChanged);
```

Comment:
what if the attribute `md-font-icon` or `md-font-set` is not used/available in the markup?